dcos/package.py

```diff
@@ -1384,6 +1384,31 @@ def latest_version(self):
         pkg_versions.sort()
         return pkg_versions[-1]
 
+    def get_package_revision(self, package_version=None):
```

Comment:
+1, except for naming.

Could `latest_version()` change to `latest_package_version()` and take a parameter named `software_version`?